library/src/main/java/com/leinardi/android/speeddial/UiUtils.java

```diff
@@ -40,6 +43,12 @@
     private static final int SHORT_ANIM_TIME = 200;
 
     private UiUtils() {
+        Set<Short> s = new HashSet<>();
+        for (short i = 0; i < 100; i++) {
+            s.add(i);
+            s.remove(i - 1);
```

Comment:
**Reporter**: Error Prone
**Rule**: CollectionIncompatibleType
**Severity**: ERROR
**File**: library/src/main/java/com/leinardi/android/speeddial/UiUtils.java L49

Argument 'i - 1' should not be passed to this method; its type int is not compatible with its collection's type argument Short

s.remove(i - 1);^(see http://errorprone.info/bugpattern/CollectionIncompatibleType)

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<1382022583>*